docs/admin/predeployment_check.md

```diff
@@ -289,6 +289,22 @@ commandline (including source builds), limits should be adjusted in
 [this article](https://access.redhat.com/solutions/61334) (which is a RHEL
 specific document but the instructions apply to most Linux distributions).
 
+### Memory mapped areas
+
+Low max number of per-process mapped areas (vm.max_map_count) can cause ULT
+stack allocation to fall-back from DAOS mmap()'ed way into Argobots preferred
+allocation method.
+
+For RPM installations, this is achieved thru installed
+`/etc/sysctl.d/10-daos-server.conf` file.
+
+For non-RPM installations, this may need to be bumped (usual default of
+65530 is too low for non-testing configurations), and the best way to do
+so is to copy `utils/rpms/10-daos_server.conf` into `/usr/lib/sysctl.d/`
```

Comment:
```suggestion
so is to copy `utils/rpms/10-daos_server.conf` into `/etc/sysctl.d/`
```

`/usr/lib/*` is reserved for files installed by packages.  `/etc/*` is the user-configured equivillent/override location where users/admins can make local configuration changes.

Moreover, files in `/usr/lib/*` will/should not be marked as configuration files in RPMs so a subsequent RPM installation would overwrite what the user added.  Files in `/etc` will be marked as configuration changes so that RPMs will not overwrite their contents but instead will create a `.rpmnew` allowing the user to compare what the package wanted to put into the file with what they have in it.